django/cantusdb_project/main_app/signals.py

```diff
@@ -170,3 +175,10 @@ def generate_volpiano_intervals(volpiano_notes):
         volpiano_intervals=volpiano_intervals,
     )
 
+def update_prefix_field(instance):
+    pk = instance.pk
+
+    prefix = str(instance.feast_code)[0:2]
```

Comment:
Can you check whether this works as expected when a feast gets saved with an empty `feast_code`?